src/components/TradeWidget/TxMessage.tsx

```diff
@@ -30,25 +31,49 @@ export const TxMessage: React.FC<TxMessageProps> = ({ sellToken, receiveToken })
 
   return (
     <TxMessageWrapper>
-      <p>
-        Your are selling {sellTokenAmount} {displaySellToken} for {displayReceiveToken}
-      </p>
-      <p>At a price of</p>
-      <p>
-        {priceInverse} {displaySellToken} per {displayReceiveToken}
-      </p>
-      <p>
-        {price} {displayReceiveToken} per {displaySellToken}
-      </p>
-      <p>
-        <strong>
-          You will receive at least {receiveTokenAmount} {displayReceiveToken}
-        </strong>
-      </p>
-      <p>
-        Your order starts {formatTimeInHours(validFrom || 0, 'now')}{' '}
-        {validUntil && `and will expire ${formatTimeInHours(validUntil, 'Never')}`}
-      </p>
+      <div className="intro-text">
+        <div>Carefully review the information below to make sure everything looks correct.</div>
+      </div>
+      <div className="message">
+        {/* Details */}
+        <div>
+          <strong>Order Details</strong>
+        </div>
+        <div>
+          Sell: <span>{sellTokenAmount}</span> <strong>{displaySellToken}</strong> for{' '}
```

Comment:
I would remove the "for GUSD" you can easily see this in the next line, where we show the amount of GUSD you may get